main.go

```diff
@@ -3,17 +3,20 @@ package main
 import (
 	"fmt"
 	"os"
-	"strings"
 
 	"github.com/bfirsh/whalebrew/cmd"
 )
 
 func main() {
-	// HACK: if first argument starts with "/", prefix the subcommand run.
-	// This allows us to use this command as a shebang, because we can't pass
-	// the argument "run" in the shebang on Linux.
-	if len(os.Args) > 1 && strings.HasPrefix(os.Args[1], "/") {
-		cmd.RootCmd.SetArgs(append([]string{"run"}, os.Args[1:]...))
+
+	if len(os.Args) > 1 {
+		// Check if not command exists
+		if _, _, err := cmd.RootCmd.Find(os.Args); err != nil {
+			// Check if file exists
+			if _, err := os.Stat(os.Args[1]); err == nil {
```

Comment:
This seems even more flimsy than my hack. ;) e.g. I can envisage lots of stuff unexpectedly not working if they collide with command names.

You could just run `whalebrew run [pkg]` in the batch script.